modules/pool/pool.resolvers.ts

```diff
@@ -91,16 +91,21 @@ const balancerResolvers: Resolvers = {
             }
             return poolService.getGqlLinearPools(chains);
         },
-        poolGetGyroPools: async () => {
-            return poolService.getGqlGyroPools();
+        poolGetGyroPools: async (parent, { chains }, context) => {
+            const currentChain = headerChain();
+            if (!chains && currentChain) {
+                chains = [currentChain];
+            } else if (!chains) {
+                throw new Error('poolGetGyroPools error: Provide "chains" param');
```

Comment:
don't we want to make chain optional? adding type parameter to the pools query might save us some complexity as well.